b.txt

```diff
@@ -0,0 +1,2 @@
+PR Commit 1
+PR Commit 2
```

Comment:
Does this update in the editor once changes are made?

Reply:
Okay, so you need to actually change an existing line (not just inserting something new or removing whitespace to get the outdated).